css/main.css

```diff
@@ -325,3 +325,48 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact_form_section {
+  height: 100vh;
+  display: flex;
+}
+
+.contact_form_header {
+  max-width: 20vw;
+  margin: 50px 30px 0 10vw;
+}
```

Comment:
- [ ] **[REQUIRED CHANGES]** Try to use a `sans-serif` font-family on the `.contact_form_header`, so it can match the Figma Template. I recommend you use the same font as the rest of the page, to increase the users' confidence in the page. A design that isn't consistent can give the impression of something fake, or of low quality, and hard your/your clients' trustfulness!